docs/common-tasks/incidents.md

```diff
@@ -0,0 +1,468 @@
+# View and Maintain Incident Records
+
+In this example we'll show you how to view incidents from your console, download them as a json file, followed by acknowledging and deleting them from your console.
+
+Before we get stuck in, it's worth mentioning that we have a host of [Useful Scripts](https://github.com/thinkst/canary-utils) that are worth looking into.
```

Comment:
Can we move this to the `Overview` page for `Common Examples`.

We can then change it to something like:

We also have a collection of [useful scripts](https://github.com/thinkst/canary-utils) that we've either given to, or had a customer give to us. You may find them useful to use in conjunction with the following examples.